datalad/interface/run_procedure.py

```diff
@@ -151,6 +199,25 @@ class RunProcedure(Interface):
     of taking at least one positional argument (the absolute path to the
     dataset they shall operate on).
 
+    For further customization there are two configuration settings per procedure
+    available:
+
+    - 'datalad.procedures.<NAME>.call-format'
+      fully customizable format string to determine how to execute procedure
+      NAME (see also datalad-run).
+      It currently requires to include the following placeholders:
```

Comment:
s/currently requires to/must/

Though I suppose only "{script}" is strictly necessary.  I imagine all scripts would want the dataset, but `args` may often be empty and a callfmt without `{args}` would work fine.

Reply:
Actually this requires an addition. While `{args}` isn't actually necessary, I think it would currently crash if it isn't in the format string, but one gives additional args when calling `run-procedure`. This should yield a proper error instead.

Reply:
`.format` works fine if a placeholder isn't present, so I suspected it would be OK, and I tested it locally and it worked fine, but perhaps something was wonky with my test.